src/coreclr/src/vm/eventpipeprotocolhelper.h

```diff
@@ -43,14 +41,17 @@ struct EventPipeCollectTracing2CommandPayload
     bool rundownRequested;
     CQuickArray<EventPipeProviderConfiguration> providerConfigs;
     static const EventPipeCollectTracing2CommandPayload* TryParse(BYTE* lpBuffer, uint16_t& BufferSize);
+
+    ~EventPipeCollectTracing2CommandPayload()
+    {
+        providerConfigs.Destroy();
```

Comment:
Is this necessary? CQuickArray has destructor that should take care of it.

Reply:
You're correct.  I somehow missed the constructor when I was looking at the class.